app/templates/navbar.html

```diff
@@ -14,7 +14,10 @@
         <div id="navbar-collapse-1" class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
                 {% for child in children %}
-                <li><a href="{{ child.attr.redirect_url|default:child.get_absolute_url }}">{{ child.get_menu_title }}</a></li>
+                    <!-- Delete this if statement on the due date -->
+                    {% if child.get_menu_title != "Kandidaten und Kandidatinnen" and child.get_menu_title != "Candidats et candidates" and child.get_menu_title != "Candidati e candidate" %}
```

Comment:
If you can't do it from the CMS, maybe it'd be better to match the URL instead

Reply:
I have changed this.